packages/frontend-api/src/Resources/config/graphql-types/PriceDecorator.types.yml

```diff
@@ -0,0 +1,20 @@
+PriceDecorator:
+    type: object
+    config:
+        description: "Represents the price of the product"
+        fields:
+            id:
```

Comment:
there should be uuid or nothing

Reply:
Rermoved id - https://github.com/shopsys/shopsys/pull/1471/commits/be6807b0ef19133f863696f2364ef00c26e2a369